bundles/org.openhab.binding.tibber/README.md

```diff
@@ -0,0 +1,119 @@
+
+## Supported Things
+
+Provided one have Tibber Account, the Tibber API is recognized as a thing in OpenHab using the Tibber binding. Tibber Pulse is optional, but will enable live measurements. The channels (i.e. measurements) associated with the binding: 
+
+Tibber Account:
+
+| Channel ID         | Description                             | Read-only |
+|--------------------|-----------------------------------------|-----------|
+| Current Total      | Current Total Price (energy + tax)      | True      |
+| Starts At          | Current Price Timestamp                 | True      |
+| Daily Cost         | Daily Cost (last/previous day)          | True      |
+| Daily Consumption  | Daily Consumption (last/previous day)   | True      |
+| Daily From         | Timestamp (daily from)                  | True      |
+| Daily To           | Timestamp (daily to)                    | True      |
+| Hourly Cost        | Hourly Cost (last/previous hour)        | True      |
+| Hourly Consumption | Hourly Consumption (last/previous hour) | True      |
+| Hourly From        | Timestamp (hourly from)                 | True      |
+| Hourly To          | Timestamp (hourly to)                   | True      |
+
+Tibber Pulse:
+
+| Channel ID              | Description                              | Read-only |
+|-------------------------|------------------------------------------|-----------|
+| Timestamp               | Timestamp for live measurements          | True      |
+| Power                   | Live Power Consumption                   | True      |
+| Last Meter Consumption  | Last Recorded Meter Consumption          | True      |
+| Accumulated Consumption | Accumulated Consumption since Midnight   | True      |
+| Accumulated Cost        | Accumulated Cost since Midnight          | True      |
+| Currency                | Currency of Cost                         | True      |
+| Min Power               | Min Power Consumption since Midnight     | True      |
+| Average Power           | Average Power Consumption since Midnight | True      |
+| Max Power               | Max Power Consumption since Midnight     | True      |
+| Voltage 1-3             | Voltage per Phase                        | True      |
+| Current 1-3             | Current per Phase                        | True      |
+| Power Production        | Live Power Production                    | True      |
+| Accumulated Production  | Accumulated Production since Midnight    | True      |
+| Min Power Production    | Min Power Production since Midnight      | True      |
+| Max Power Production    | Max Power Production since Midnight      | True      |
+
+
+## Binding Configuration
+
+To access and initiate the Tibber binding, a Tibber account is required.
+
+Required input needed for initialization:
```

Comment:
It's easier to read when writing it like

The following input is required for initialization:

Reply:
Corrected.